rancher/latest/en/kubernetes/index.md

```diff
@@ -48,6 +48,9 @@ In the config file, it provides the necessary information for your local machine
 
 Rancher provides a convenient shell access to a managed kubectl instance that can be used to manage Kubernetes clusters and applications. 
 
+### Adding a Private Registry To Kubernetes
+
+Private registries can be used with Kubernetes services by adding your [private registry]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/configuration/registries/) in all of your Kubernetes environments. 
```

Comment:
"in all of your Kubernetes environments" sounds a bit confusing. Can we just say that you should add the private registry to the kubernetes environment in case you need to use it in a particular environment? something like that